userspace/libsinsp/eventformatter.cpp

```diff
@@ -163,6 +163,42 @@ bool sinsp_evt_formatter::on_capture_end(OUT string* res)
 	return res->size() > 0;
 }
 
+bool sinsp_evt_formatter::resolve_tokens(sinsp_evt *evt, map<string,string>& values)
+{
+	bool retval = true;
+	const filtercheck_field_info* fi;
+
+	ASSERT(m_tokenlens.size() == m_tokens.size());
+
+	uint32_t j = 0;
```

Comment:
minor style thing: this would probably fit better together with the other variables declaration at L168 or embedded in the `for` loop.